src/components/events/ManageTicketTiers.tsx

```
@@ -0,0 +1,275 @@
+          .order("start_date", { ascending: true, nullsFirst: false });
+
+        if (error) throw error;
+        setTiers(
+          (data || []).map((t) => ({
+            id: t.id,
+            name: t.name,
+            price: t.price / 100, // convert cents to dollars
+            capacity: t.capacity,
+            start_date: t.start_date ? t.start_date.slice(0, 16) : "", // format for datetime-local
+            end_date: t.end_date ? t.end_date.slice(0, 16) : "",
+          }))
```

ESLint

[error] Insert `,`

(prettier/prettier)